scrapi/harvesters/bhl.py

```diff
@@ -0,0 +1,64 @@
+"""Harvests Biodiversity Heritage Library OAI Repository (BHL) metadata for ingestion 
```

Comment:
W291 trailing whitespace
Python style guidelines recommend that you trim trailing whitespace at the end of lines